hw/ip/kmac/data/kmac.prj.hjson

```diff
@@ -12,10 +12,10 @@
       {
         version:            "1.0",
         life_stage:         "L1",
-        design_stage:       "D2",
+        design_stage:       "D2S",
         verification_stage: "V2",
         dif_stage:          "S1",
-        commit_id:          "862155839",
+        commit_id:          "54b949255a8c463ea365126d90c39cd26c462638",
```

Comment:
Is this a valid commit ID?

If you don't want to maintain the commit ID here, I believe it is fine to remove it.
In other IPs we only added the commit ID when we had to revert the version to a previous design stage, e.g.:
https://github.com/lowRISC/opentitan/blob/c3dc1fa938d1889e806ff531be305afcaeb52ad8/hw/ip/gpio/data/gpio.prj.hjson#L12-L27

Reply:
Yep. That is the last commit that @m-temp pushed. I normally specified the commit id when the design stage has been changed.

Reply:
Ah ok sounds good. I only looked at your last two commits in this PR since I thought you were referring to one of them - and they did not match :).

Reply:
shouldn't we include your (two) fixes? So if we ever want to go back, they are covered?

Reply:
Yeah that is a good point. I would either do what @m-temp suggested, or remove the id entirely.